modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java

```diff
@@ -357,7 +346,12 @@ private ObjectField _toObjectField(
 				).put(
 					"update",
 					() -> {
-						if (!updateable) {
+						if ((objectDefinition.isApproved() ||
```

Comment:
I did not understand why objectDefinition.isApproved() need to be in this condition. Could you explain, please?

Reply:
@marcelabc I was just keeping the original behavior from [here](https://github.com/liferay-objects/liferay-portal/blob/6221e884ae62e693f52fb358482185dd16383216/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java#L323-L328). 

I believe this means that we cannot update object fields that were created before publishing the object definition.

Reply:
Thanks!